src/Components/Playlist/PlayList.js

```diff
@@ -0,0 +1,16 @@
+import TrackList from './TrackList';
+import './PlayList.class';
+
```

Comment:
```suggestion
import React from "react";
```

Reply:
I would say this has been resolved --ej--